developers.libra.org/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contribution Guide
```

Comment:
Might want to merge this with the libra one?

Reply:
I added a line in the main `CONTRIBUTING.md` file that talks a bit about documentation and then points to this `CONTRIBUTING.md` -- since this one is kinda long, I didn't want to overwhelm the main one.